airflow/plugins/operators/sentry_to_gcs.py

```diff
@@ -133,12 +133,11 @@ def fetch_and_clean_from_sentry(extract, target_date, auth_token):
 class SentryExtract(PartitionedGCSArtifact):
     bucket: ClassVar[str] = CALITP_BUCKET__SENTRY_EVENTS
     table: ClassVar[str] = "events"
-    dt: pendulum.Date
-    ts: pendulum.DateTime
-    partition_names: ClassVar[List[str]] = ["dt", "ts"]
+    execution_dt: pendulum.Date
+    execution_ts: pendulum.DateTime
```

Comment:
I think `dt` is still an OK name, since it's the data's actual date. `execution_ts` is confusing because it's set to a different value within the task, but the filename uses the value from the Airflow context. Is it intended to be an identifier of the task instance or the actual wall clock time?